### ISSUE TITLE
What cause the large performance improvement on 3DPW dataset?

### ISSUE THREAD

**Issue by linjing7** [HongwenZhang/PyMAF#58]
Hi, thank you very much for your excellent work.

In the journal version, the performance of PyMAF is significantly better than that in the conference version. Could you please tell me the reason? Is it caused by the differences between the training setting?

**Comment by HongwenZhang** [comment]
Hi, thanks for your questions. 

Yes, it is caused by the differences between the training settings used in the conference version (following the setting in [SPIN](https://github.com/nkolot/SPIN)) and the journal version (following the setting in [PARE](https://github.com/mkocabas/PARE)). Basically, the performance of the baseline (HMR architecture) is significantly better than the conference version when following the same training setting as [PARE](https://github.com/mkocabas/PARE). In comparison with SPIN, the training setting of PARE mainly differs in the following aspects:

1. Using the [EFT](https://github.com/facebookresearch/eft) pseudo ground-truth labels.
2. Using the weights pretrained in [pose_resnet](https://github.com/Microsoft/human-pose-estimation.pytorch) for a 2D pose estimation task to initialize the backbone network.
3. Using different dataset ratios (i.e. first 100% COCO-EFT then mixture datasets) in the training.

More details about the training of PARE can be found in its [paper](https://arxiv.org/abs/2104.08527):
<img width="865" alt="image" src="https://user-images.githubusercontent.com/12066626/185268292-10f7fd59-3c15-4f88-856e-2a618f8d3a1f.png">

The issues of the numerical evaluation and its fairness are also discussed in our [survey paper](https://github.com/tinatiansjz/hmr-survey).

**Comment by linjing7** [comment]
Okay, thank you very much for your reply!
